mm-bot/docs/README.md

```diff
@@ -0,0 +1,91 @@
+# MM Bot
+MM Bot is a process designed to supply liquidity to YAB Escrow orders.
+
+## Logical View
+### Functional Requirements
+- The bot must be able to read an order from the Escrow contract.
+- The bot must be able to perform a transfer in Ethereum to the recipient address through the Payment
+Registry contract.
+- The bot must be able to perform a repay in Ethereum to recover the funds in the L2 through the
+Payment Registry contract.
+- The bot must be able to store the orders in a database and update their status.
+- In case of an error, the bot must be able to store the error and retry the order.
+
+### Simplified Class Diagram
+The following diagram shows the mm classes and how they interact with each other.
+
+![mm_diagram_class.svg](images%2Fmm_diagram_class.svg)
+
+### Full Class Diagram
+The following diagram is a detailed version of the previous diagram, 
+showing the attributes and methods of each class.
+
+![mm_diagram_class_full.svg](images%2Fmm_diagram_class_full.svg)
```

Comment:
this diagram can't be appreciated in the context where it will be read by external users (which is directly from github)